.lighthouserc-accessibility-2.js

```diff
@@ -63,7 +63,9 @@ module.exports = {
         },
         {
           'matchingUrlPattern': '^http://127.0.0.1:8181/volunteer$',
-          'assertions': baseConfig['baseAccessibilityAssertions']
+          'assertions': {
+            'categories:accessibility': ['error', {'minScore': 0.9}]
```

Comment:
After upgrading the webdriver version, this score went down. I suspect new checks were added to calculate this score. See the report here: https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1628909640729-82260.report.html#accessibility

For fixing the ARIA issue in `ngbCarosel` (invalid aria `aria-activedescendant`), the ng-bootstrap library may need to be updated per https://github.com/ng-bootstrap/ng-bootstrap/issues/4014. This is not in the scope of this PR. Created TODO for increasing the accessibility score (https://github.com/oppia/oppia/issues/13668)